Mathlib/Order/Hom/Basic.lean

```diff
@@ -898,6 +898,8 @@ theorem toEquiv_symm (e : α ≃o β) : e.toEquiv.symm = e.symm.toEquiv :=
   rfl
 #align order_iso.to_equiv_symm OrderIso.toEquiv_symm
 
+theorem coe_symm (e : α ≃o β) : (e : α ≃ β).symm = e.symm := rfl
```

Comment:
Is this not syntactically the same theorem as above? If so, can you instead tag the above with `norm_cast`?

Reply:
`toEquiv_symm` can't be tagged with `norm_cast` because it doesn't contain any coercions. `coe_symm` on the other hand can.

Reply:
Ah sorry this is the coercion mess with hom classes. I need to sort this out.